tests/TestHelpers/OcmHelper.php

```diff
@@ -0,0 +1,108 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Kiran Parajuli <kiran@jankaritech.com>
+ * @copyright Copyright (c) 2022 Kiran Parajuli kiran@jankaritech.com
+ */
+
+namespace TestHelpers;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * A helper class for managing Graph API requests
+ */
+class OcmHelper {
+	/**
+	 * @return string[]
+	 */
+	private static function getRequestHeaders(): array {
+		return [
+			'Content-Type' => 'application/json',
+		];
+	}
+  
+	/**
+	 * @param string $baseUrl
+	 * @param string $path
+	 *
+	 * @return string
+	 */
+	public static function getFullUrl(string $baseUrl, string $path): string {
+		$fullUrl = $baseUrl;
+		if (\substr($fullUrl, -1) !== '/') {
+			$fullUrl .= '/';
+		}
+		$fullUrl .= 'sciencemesh/' . $path;
+		return $fullUrl;
+	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string|null $email
+	 * @param string|null $description
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function createInvitation(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		?string $email = null,
+		?string $description = null
+	): ResponseInterface {
+		$body['params'] = [
+		  "description" => $description,
+		  "recipient" => $email
+		];
+		$url = self::getFullUrl($baseUrl, 'generate-invite');
+		return HttpRequestHelper::post(
+			$url,
+			$xRequestId,
+			$user,
+			$password,
+			self::getRequestHeaders(),
+			\json_encode($body)
+		);
+	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $token
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function acceptInvitation(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		string $token
+	): ResponseInterface {
+		$body = [
+		  "token" => $token,
+		  "providerDomain" => 'localhost'
```

Comment:
is `localhost` correct?

Reply:
ah thanks, should be `ocis-server` for CI and `localhost` for local running.
I would take care of this in the next PR and add documentation on how to run ocm tests.